extra/macos_pkg_files/readme.txt

```diff
@@ -0,0 +1,5 @@
+OpenOrbis PS4 Toolchain will be installed at /opt/OpenOrbis-PS4-Toolchain and the installer will automatically set the OO_PS4_TOOLCHAIN environment variable for you in both bash (Mojave and lover) and zsh (Catalina) shells. If you use another shell you will need to set the environment variable for yourself.
```

Comment:
This can probably be added to the readme.md instead of having a second readme.txt, kind of confusing.

Reply:
I didn't want to leave the installer's readme empty nor I wanted to include the full readme, so that's why I wrote this little readme with macOS-specific info

I can add this info in the main readme too